README.md

```diff
@@ -9,9 +9,9 @@
 
 Build | Status
 -- | --
-Ubuntu Jammy  | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gz_cmake-ci-main-jammy-amd64)](https://build.osrfoundation.org/job/gz_cmake-ci-main-jammy-amd64)
-Homebrew      | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gz_cmake-ci-main-homebrew-amd64)](https://build.osrfoundation.org/job/gz_cmake-ci-main-homebrew-amd64)
-Windows       | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gz_cmake-main-win)](https://build.osrfoundation.org/job/gz_cmake-main-win)
+Ubuntu Jammy  | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gz_cmake-ci-gz-cmake4-jammy-amd64)](https://build.osrfoundation.org/job/gz_cmake-ci-gz-cmake4-jammy-amd64)
```

Comment:
```suggestion
Ubuntu Noble  | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gz_cmake-ci-gz-cmake4-noble-amd64)](https://build.osrfoundation.org/job/gz_cmake-ci-gz-cmake4-noble-amd64)
```

Reply:
Done in https://github.com/gazebosim/gz-cmake/pull/445/commits/8dbfb98319abc6cef0f97bfd98928b27529c8dca